scripts/update_contributors.py

```
@@ -0,0 +1,123 @@
+    page = 1
+    while True:
+        query = urlencode({"per_page": 100, "page": page})
+        request = Request(
+            f"{GITHUB_API}/repos/{repository}/contributors?{query}",
+            headers={
+                "Accept": "application/vnd.github+json",
+                "User-Agent": "skillsaw-contributor-updater",
+                "X-GitHub-Api-Version": "2022-11-28",
+                **({"Authorization": f"Bearer {token}"} if token else {}),
+            },
+        )
```

Ruff (0.16.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)